### PR TITLE
add connors-hello-world-app

### PR DESCRIPTION
**Pull request by connormaglynn**

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/connors-hello-world-app/01-rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: connors-hello-world-app-admin
+  namespace: connors-hello-world-app
+subjects:
+  - kind: Group
+    name: "github:connormaglynn"
```
Comment (vijay-veeranki) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/connors-hello-world-app/01-rbac.yaml:8]: This should be the github team name, for ex: check-my-diary

**Review by vijay-veeranki — CHANGES_REQUESTED** [review]
@connormaglynn please update the team name in rbac.yaml file
subjects:
  - kind: Group
    name: "github:connormaglynn"

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/connors-hello-world-app/01-rbac.yaml (hunk 1 above)
Comment (connormaglynn) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/connors-hello-world-app/01-rbac.yaml]: created a new team of @ministryofjustice/connors-team (trying to keep it clear that's it's just testing 😄 )

**Review by vijay-veeranki — APPROVED** [review]

**Comment by vijay-veeranki** [comment]
@connormaglynn do you still need this PR?

**Pull request closed without merging**